src/test/java/org/broadinstitute/hellbender/tools/IntegrationTestSpec.java

```diff
@@ -123,12 +122,8 @@ private void executeTest(String testName, CommandLineProgramTest testClass, Stri
         boolean gotAnException = false;
         try {
             final String now = LocalDateTime.now().format(DateTimeFormatter.ofPattern("HH:mm:ss"));
-            final String cmdline = String.join(" ", command);
-            System.out.println(String.format("[%s] Executing test %s:%s with GATK arguments: %s", now, testClass.getClass().getSimpleName(), testName, cmdline));
-            // also write the command line to the HTML log for convenient follow-up
-            // do the replaceAll so paths become relative to the current
-            BaseTest.log(cmdline.replaceAll(BaseTest.publicTestDirRoot, ""));
-            Object result = testClass.runCommandLine(command);
+            System.out.println(String.format("[%s] Executing test %s:%s", now, testClass.getClass().getSimpleName(), testName));
```

Comment:
@droazen If we set it to logger.info() it will always be suppressed.  It's only used for outputting info during tests and by default we don't print info in tests.  If you think it's not useful we can remove it entirely.